.github/workflows/wheels.yaml

```diff
@@ -0,0 +1,60 @@
+name: Build CI Wheels
+
+on:
+  push:
+    branches:
+      - main
+      - stable
+    tags:
+      - yt-*
+
+jobs:
+  build_wheels:
+    name: Wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+      fail-fast: false
+
+    steps:
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: 3.9
+
+      - uses: s-weigand/setup-conda@v1
+        if: matrix.os == 'windows-latest'
+        with:
+          update-conda: true
+          conda-channels: conda-forge
+          activate-conda: true
+          python-version: 3.9
+
+      - uses: actions/checkout@v2
+      - name: Install cibuildwheel
+        run: python -m pip install cibuildwheel==1.9.0
+
+      - name: Install dependencies and yt
+        shell: bash
+        env:
+          dependencies: "full"
+          LDFLAGS: "-static-libstdc++"
+        run: source ./tests/ci_install.sh
+
+      - name: Build wheels for CPython
+        run: |
+          python -m cibuildwheel --output-dir dist
+        env:
+          CIBW_BUILD: "cp37-* cp38-* cp39-*"
```

Comment:
Is Python 3.6 left out intentionally ?

Reply:
I thought it's EOL already, but apparently that happens only in 4 months...

Reply:
FYI, we'll drop support for Python 3.6 on _next_ release, see #2917